### DISCUSSION TITLE
How to use this feature

### DISCUSSION THREAD

**Discussion by vaisakhc56** [anasty17/mirror-leech-telegram-bot#835] [Q&A]
![IMG_20220731_120048](https://user-images.githubusercontent.com/64364212/182013365-07647074-1cfe-46c8-92c8-a56a9563d59e.jpg)
![Uploading IMG_20220731_120048.jpg…]()

How to use this feature

**Comment by anasty17** [comment]
Fork and give star then ask this answered question!

**Comment by anasty17** [comment]
Send links to the chats then reply to first link by the desired command.
